source/slang/core.meta.slang

```diff
@@ -602,6 +614,14 @@ ${{{{
 }}}}
     ,  __BuiltinArithmeticType
     ,  __BuiltinIntegerType
+${{{{
+    if (kBaseTypes[tt].tag == BaseType::Int || kBaseTypes[tt].tag == BaseType::UInt)
+}}}}
+    ,  __BuiltinInt32Type
```

Comment:
I don't know how the `${{{{` magic works.
But it will be safer to wrap this in a block like,
```
if (kBaseTypes[tt].tag == BaseType::Int || kBaseTypes[tt].tag == BaseType::UInt)
{ // <=== ADD THIS ONE
${{{{
    , __BuiltinInt32Type
}}}}
} // <== ADD THIS ONE TOO

Reply:
every line not in ${{{ wil map to:
```
sb << "line of text\n";
```

So 
```
${{{
   if (blah)
}}}
line of text
```
maps to:
```cpp
if (blah)
     sb << "line of text\n";
```